package.json

```diff
@@ -29,7 +29,8 @@
     "onCommand:vscode-ansible.cloudshell",
     "onCommand:vscode-ansible.playbook-in-docker",
     "onCommand:vscode-ansible.playbook-in-localansible",
-    "onCommand:vscode-ansible.ssh"
+    "onCommand:vscode-ansible.ssh",
+    "onCommand:vscode-ansible.deployment-templates"
```

Comment:
azure-deployment-templates? need to specify it's azure specific

Reply:
i think it will no longer be azure specific if customized as discussed below

Reply:
which module will be used for User xxx templates? I suppose it still be azure_rm_deployment module, right? or any module could use xxx templates? If it's azure_rm_deployment, then it's Azure specific

Reply:
haha, you're right, there are no non-Azure ARM templates :-)